### ISSUE TITLE
Change wpull args during a crawl

### ISSUE THREAD

**Issue by Coloradohusky** [ArchiveTeam/grab-site#168]
As the title says, is there any way to change/add wpull args during a crawl?

**Comment by ivan** [comment]
No, unless you 1) modify grab-site's wpull hook to do something extra, or 2) really want to connect to the manhole and hack away at some internal wpull state

What did you want to change?

**Comment by Coloradohusky** [comment]
I want to add `--retry-connrefused`, as I don't want to miss any pages, and I don't want to start over, cause I've been running my crawl for about a week or so now lol
